tutor/src/components/onboarding/free-trial-activated.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import { observer, PropTypes as MobxPropTypes } from 'mobx-react';
+import { OnboardingNag, Heading, Body, Footer } from './onboarding-nag';
+import CourseUX from '../../models/course/ux';
+
+@observer
+export default class FreeTrialActivated extends React.PureComponent {
+
+  static propTypes = {
+    ux: MobxPropTypes.observableObject.isRequired,
+  }
+
+  static className = 'free-trial-activated';
```

Comment:
yeah, that's because the on boarding components are wrapped inside the on boarding modal.  Some of them need to set the class on the outer modal, so the on boarding reads the static prop here so it can set it.